src/algorithm/contains.rs

```diff
@@ -0,0 +1,239 @@
+
+pub use types::{Coordinate, Point, LineString, Polygon, MultiPolygon};
```

Comment:
I think you shouldn't put `pub` in your uses if you don't want to forward import them